package.json

```diff
@@ -23,7 +24,8 @@
     "mocha": "^8.2.1",
     "sinon": "^9.0.1",
     "ts-node": "^9.1.0",
-    "typescript": "^4.1.2"
+    "typescript": "^4.1.2",
+    "lighthouse": "https://github.com/GoogleChrome/lighthouse.git#master"
```

Comment:
FYI this doesn't actually get the _latest_ code, but it pinned in yarn.lock. If you want the latest, we can do yarn add -D in GH actions.

If using the latest doesn't particularly matter, then let's just pin to 9.6.1

Reply:
hmm yarn add -D in the yml would actually be preferable to me...  yah good call about the lock. 

i'll switch to this.